weave/flow/eval.py

```diff
@@ -268,8 +271,14 @@ async def summarize(self, eval_table: list) -> dict:
                     scorer_name, _, summarize_fn = get_scorer_attributes(scorer)
                     scorer_stats = transpose(vals)
                     score_table = scorer_stats[scorer_name]
-                    scored = summarize_fn(score_table)
-                    summary[scorer_name] = scored
+
+                    if errs := [
+                        s for s in score_table if isinstance(s, dict) and "error" in s
+                    ]:
+                        summary[scorer_name] = {"errors": errs}
```

Comment:
This reproduces the list of errors that we've already logged in each predict_and_score call. Since we're using the trace itself as the data model for evaluations, I don't think we should do this.

Reply:
See comment above -- I think this has nice properties